data/projects/rstcheck_with_better_sphinx_support.md

```diff
@@ -0,0 +1,34 @@
+[`rstcheck`](https://github.com/myint/rstcheck) is a tool that checks the
+syntax of code blocks nested within reStructuredText. Currently, the Sphinx
+support in `rstcheck` is minimal. This results in
+[false positives](https://github.com/myint/rstcheck/issues/19).
+
+It would be interesting to experiment with an alternative `rstcheck` mode that
+uses `sphinx-build` instead of `docutils` directly. This would allow `rstcheck`
+to parse Sphinx documents exactly the same way Sphinx does.
+
+* `rstcheck` = `docutils` + code block linting
+* `rstcheck-sphinx` = `sphinx-build` + code block linting
+
+#### Milestones
+
+##### GSOC 2017 CODING STARTS
+
+* Be familiar what `rstcheck` does and how it does it.
+* Be familiar with how to extend Sphinx. Particularly, learn how to parse code
+  blocks.
```

Comment:
Can you clarify what the "CODING STARTS" section means? Based on one of the other examples I looked at, it seemed like prerequisites.